api/src/wfl/module/covid.clj

```diff
@@ -523,19 +530,24 @@
     (jdbc/insert-multi! tx details (map #(-> {:item (pr-str %)}) snapshots))
     [tdr-snapshot-list-type details]))
 
-(defn ^:private load-tdr-snapshot-list [tx {:keys [source_items] :as _workload}]
+(defn ^:private load-tdr-snapshot-list
+  [tx {:keys [started source_items] :as _workload}]
   (when-not (postgres/table-exists? tx source_items)
     (throw (ex-info "Failed to load tdr-snapshot-list: no such table"
                     {:table source_items})))
-  {:type tdr-snapshot-list-type :items source_items})
+  {:type      tdr-snapshot-list-type
+   :items     source_items
+   :stopped   started
+   :snapshots (postgres/get-table tx source_items)})
 
 (defn ^:private start-tdr-snapshot-list [_ source] source)
+(defn ^:private stop-tdr-snapshot-list  [_ source] source)
 (defn ^:private update-tdr-snapshot-list [source]  source)
 
 (defn ^:private peek-tdr-snapshot-list [{:keys [items] :as _source}]
   (let [query "SELECT *        FROM %s
                WHERE  consumed IS NULL
-               ORDER BY id ASC LIMIT 1"]
```

Comment:
Is there any possibility that this deletion could lead to popping a different record than the one peeked?

Reply:
no, they're returned in id order anyway

Reply:
The reason I brought this up is that when writing tests, I found that this isn't always true.  I inserted two records and they were written with ids 1 and 2.  I updated the second record but not the first.  When I fetched the contents of the table, they were returned as `(2, …) (1, …)`.

I had to sort the records by id to get the test to pass:
https://github.com/broadinstitute/wfl/blob/develop/api/test/wfl/integration/modules/covid_test.clj#L338